Dockerfile

```diff
@@ -28,9 +28,6 @@ RUN rustup target add wasm32-unknown-unknown --toolchain nightly-2019-07-14
 # (polkadot)
 RUN cargo +nightly install --git https://github.com/alexcrichton/wasm-gc
 
-# setup default stable channel
-RUN rustup default nightly-2019-07-14
```

Comment:
Why can we remove this in release mode?

Reply:
It's a redundant command, nothing to do with release mode. 
We already set it in line 24.